File: Question 8/reverse_words.js
function reverse_words(string){
    return string.split("").reverse().join("").split(" ").reverse().join(" "); 
  }
  
  console.log(reverse_words('simplicity is prerequisite for reliability')); //yticilpmis si etisiuqererp rof ytilibailer
  
  console.log(reverse_words('keep coding')); //peek gnidoc

File: Arrays/Question 8/reverse_words.rb
def reverse_words(sent)
    words = sent.split(" ")
    new_words = []
    words.each { |word| new_words << word.reverse }
    new_sent = new_words.join(" ")
    return new_sent
  end
  
  puts reverse_words('keep coding') # => 'peek gnidoc'
  puts reverse_words('simplicity is prerequisite for reliability') # => 'yticilpmis si etisiuqererp rof ytilibailer'